servers/rendering/renderer_rd/renderer_storage_rd.cpp

```diff
@@ -4825,6 +4825,7 @@ void RendererStorageRD::particles_request_process(RID p_particles) {
 
 	if (!particles->dirty) {
 		particles->dirty = true;
+        	particles->inactive_time = 0;
```

Comment:
Indentation should use tabs, not spaces:

```suggestion
		particles->inactive_time = 0;
```